modules/imgproc/test/test_imgwarp.cpp

```diff
@@ -1909,5 +1909,43 @@ TEST(Imgproc_warpPolar, identity)
 #endif
 }
 
+TEST(Imgproc_undistort, regression_15286)
```

Comment:
Please move this test into calib3d module: https://github.com/opencv/opencv/blob/3.4.7/modules/calib3d/test/test_undistort.cpp
(`undistort()` code is in `imgproc` module for legacy reasons only, in OpenCV 4.x it is moved to `calib3d`)

Reply:
Moved